src/check-dsl.ts

```diff
@@ -239,7 +563,17 @@ export const checkDSL = (codeInEditor: string, options: ValidationOptions = {})
       typeRegex,
       relationRegex,
     );
-    basicValidateRelation(lines, reporter, parserResults, globalRelations, transformedTypes);
+
+    switch (parserResults.schemaVersion) {
```

Comment:
```suggestion
    const schemaVersion = parserResults.schemaVersion || SchemaVersion.OneDotZero;
    switch (schemaVersion) {
```

In case schema version was not set